test/RemoteDataSource.spec.tsx

```diff
@@ -93,39 +93,15 @@ describe('RemoteDateSource', () => {
             });
           });
 
-          it('Should return a payload with records 11 to 20', (done) => {
+          it('Should refresh the DataStream', (done) => {
```

Comment:
Why don't you try using two different payloads? Just to make sure it is actually refreshing the data... Think for example, when adding a new element to the grid, a refresh needs to be fired and you should be able to see your brand new record.